scripts/cart.js

```diff
@@ -0,0 +1,64 @@
+let purchaseDetail = document.querySelector('.js-purchase-detail');
+
+/* Funtion Sumar input*/
+function sumar(target) {
+  let inputElement = target.parentElement.querySelector('.js-product-add-card-number');
+  let valorActual = parseInt(inputElement.value);
+  inputElement.value = valorActual + 1;
+}
+
+/* Funtion Restar input */
+function restar(target) {
+  let inputElement = target.parentElement.querySelector('.js-product-add-card-number');
+  let valorActual = parseInt(inputElement.value);
+  if (valorActual > 1) {
+    inputElement.value = valorActual - 1;
+  };
+};
+
+purchaseDetail.addEventListener('click', (event) => {
+  console.log(event)
+  let target = event.target
```

Comment:
Si vas a usar ese nombre, puedes usar destructuring aqui y creo que serviria igual. De paso haces uso de algo mas avanzado.

Ref.: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
```suggestion
  let { target } = event
```